src/chronometer-centiseconds.js

```diff
@@ -1,37 +1,59 @@
 class Chronometer {
   constructor() {
+    this.currentTime = 0;
+    this.intervalId = null;
     // ... your code goes here
   }
 
   start(printTimeCallback) {
-    // ... your code goes here
+    this.intervalId= setInterval(()=>{
+    this.currentTime+=1
+    if (printTimeCallback){
+      printTimeCallback();
+    }
+    },10)// ... your code goes here
   }
 
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime/6000); /// ... your code goes here
   }
 
   getSeconds() {
+   return (Math.floor(this.currentTime/100))%60
+   
     // ... your code goes here
   }
 
   getCentiseconds() {
+    
+    return  this.currentTime%100 
+    
     // ... your code goes here
   }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    if (value === 0){
+      return "0"+value.toString();
+    }
+else if(value <9){
```

Comment:
Here you need to check if value is less than 10, otherwise when it gets to 0 seconds your chronometer will flash `00:9UNDEFINED`